vendor/github.com/tsg/gopacket/pcap/pcap_tester.go

```diff
@@ -1,107 +0,0 @@
-// Copyright 2012 Google, Inc. All rights reserved.
-//
-// Use of this source code is governed by a BSD-style license
-// that can be found in the LICENSE file in the root of the source
-// tree.
-
-// +build ignore
-
-// This binary tests that PCAP packet capture is working correctly by issuing
-// HTTP requests, then making sure we actually capture data off the wire.
-package main
```

Comment:
Is the removal of this file related to 1.12.1?

Reply:
It was removed by govendor when I ran `fetch`. I thought it was removed from the original repo... But now I see that it's still there. Let me look into why the tool get rid of this file.

Reply:
The reason could be that its `package main`. My first gest was that it's a test file, but it's a script.